.github/workflows/testing.yml

```diff
@@ -0,0 +1,80 @@
+name: Testing
+
+on:
+  push:
+    branches:
+      - "*"
+      - "!skipci*"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref_name }}
+
+permissions:
+  id-token: write
+  contents: read
+  actions: read
+
+jobs:
+  unit-tests:
+    name: Unit Tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: 'Setup jq'
+        uses: dcarbone/install-jq-action@v2.1.0
+        with:
+          version: '${{ inputs.version }}'
+          force: '${{ inputs.force }}'
+      - name: 'Check jq'
+        # language=sh
+        run: |
+          which jq
+          jq --version
+      - uses: actions/setup-node@v3
+        with:
+          node-version-file: ".nvmrc"
+      - uses: actions/cache@v3
+        with:
+          path: |
+            **/node_modules
+            ~/.cache/Cypress
+          key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
+      - name: set path
+        run: |
+          echo "PATH=$(pwd)/node_modules/.bin/:$PATH" >> $GITHUB_ENV
+      - name: run unit tests (frontend)
+        run: ./scripts/test-unit.sh src/app/testing/jest/frontend
+      - name: run unit tests (backend) src/app/testing/jest/frontend
+        run: ./scripts/test-unit.sh
```

Comment:
I think the script is overkill when we could just have a step for `yarn install` (and cache it) and a step for `yarn dev:cov`

Reply:
Ooh very good point. Switching.